frontend/src/App.js

```diff
@@ -23,7 +23,7 @@ function App() {
         {hasRole(currentUser, "ROLE_ADMIN") && (
           <Route exact path="/admin/users" element={<AdminUsersPage />} />
         )}
-        {hasRole(currentUser, "ROLE_ADMIN") && (
+        {hasRole(currentUser, "ROLE_PROFESSOR") && (
```

Comment:
This is a good point—it all depends on what the intent of the target page is.

There are multiple feasible designs.  One option is a that there's a page for competed requests and one for pending requests, and what you see on that page depends on your role.   

Another design is that there are separate pages intended for students to create requests, and professors to review and respond to requests.  Those are both legitimate options.   

The important thing is to be clear with the intentions for the page, and most importantly, that the frontend and backend are *consistent* about the design choices.   That last part is what's not true now: the frontend is restricting the page to the ADMIN role while the backend is using the PROFESSOR role, which creates weird errors when an ADMIN views the page.

Reply:
Good thinking! I have now allowed students to see the completed requests as well. It simply hits the `/api/recommendationrequest/requester/all` endpoint instead of the `/api/recommendationrequest/professor/all` endpoint if the role is student.

Reply:
Currently waiting for Dokku to rebuild... 😗🎶

Reply:
OK, it's up! Feel free to test it if and when you get the chance. Thanks in advance!